walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/SortByStartTime.java

```diff
@@ -0,0 +1,9 @@
+package com.google.sps;
+
+import java.util.Comparator;
+
+public class SortByStartTime implements Comparator<Event> {
+  public int compare(Event e1, Event e2) {
+    return TimeRange.ORDER_BY_START.compare(e1.getWhen(), e2.getWhen());
+  }
+}
```

Comment:
Rather than putting this in a new file, you could add it to the `Event` class, like the Comparator in `TimeRange`.

Reply:
I think the methods in `TimeRange` can only be used to compare two `TimeRange` objects. Whereas, I want to sort the whole `List` using `Collections.sort`. 

For that you need the second argument of sort to be an object of a class that implements `Comparator<T>` (where T is type of the objects in the list you want to sort)

Reply:
Indeed :)